frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -0,0 +1,74 @@
+package com.virtuslab.gitmachete.frontend.actions.dialogs
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.ui.layout.buttonGroup
+import com.intellij.ui.layout.panel
+import com.virtuslab.gitmachete.backend.api.IGitMacheteBranch
+import com.virtuslab.gitmachete.backend.api.IGitMacheteNonRootBranch
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.getString
+import java.text.MessageFormat.format
+
+enum class OverrideOption {
+  PARENT,
+  INFERRED
+}
+
+class OverrideForkPointDialog
+    constructor(
+        project: Project,
+        private val parentBranch: IGitMacheteBranch,
+        private val branch: IGitMacheteNonRootBranch
+    ) : DialogWrapper(project, /* canBeParent */ true) {
+
+  private var myOverrideOption = OverrideOption.PARENT
+
+  init {
+    title =
+        getString("action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.title")
+    setOKButtonMnemonic('O'.toInt())
+    init()
```

Comment:
Whoops that's unfortunate ;D maybe drop a comment that a method just happens to be named this way... or qualify `DialogWrapper` explicitly if possible